values/src/main/java/org/spine3/time/LocalTimes.java

```diff
@@ -35,7 +35,7 @@ private LocalTimes() {
     }
 
     /**
-     * Obtains the current local time .
+     * Obtains current LocalTime instance.
```

Comment:
The previous comment was better. Maybe without a definite article, but still. We'd better avoid repeating class name in the documentation. It's already visible in the constructor signature.